install/resources/upgrade49.php

```diff
@@ -35,8 +35,8 @@ function upgrade49_dbchanges()
 
 	$db->delete_query("settings", "name='allowyahoofield'");
 
-	$db->modify_column('attachments', 'filename', 'varchar(255)', true, "''");
 	$db->modify_column('attachments', 'attachname', 'varchar(255)', true, "''");
+	$db->modify_column('attachments', 'filename', 'varchar(255)', true, "''");
```

Comment:
```suggestion
	$db->modify_column('attachments', 'filename', 'varchar(255)', true, "''");
	$db->modify_column('attachments', 'attachname', 'varchar(255)', true, "''");
```

Looks like this got accidentally reordered in the previous commit. You should be able to apply this suggestion to make the change. :-)